app/src/main/java/com/example/perfectweatherallyear/utils/NotificationHandler.kt

```diff
@@ -9,14 +9,20 @@ import androidx.core.app.NotificationManagerCompat
 import com.example.perfectweatherallyear.MainActivity
 import com.example.perfectweatherallyear.R
 
-const val NOTIFICATION_ID = 1
-const val CHANNEL_ID  = "reminder_channel_id"
+const val START_NOTIFICATION_SERVICE_ID = 1
+const val NOTIFICATION_ID = 2
+const val CHANNEL_ID = "reminder_channel_id"
+
 class NotificationHandler {
 
     companion object {
-        fun createNotification(application: Application, context: Context, contentMessage: String): Notification {
+        fun createNotification(
```

Comment:
fun named createNotification, but inside it is also post notification - bad practice.